guides/common/modules/proc_migrating-a-host-to-use-pull-mode.adoc

```diff
@@ -0,0 +1,39 @@
+[id="migrating-a-host-to-use-pull-mode_{context}"]
+= Migrating a Host to Use Pull Mode
+
+To migrate a host from `katello-agent` to pull-based transport, you can use the `katello-pull-transport-migrate` utility.
+
+.Prerequisites
+* The host runs an Enterprise Linux derivative operating system.
+ifndef::satellite[]
+The utility should work on any system that uses Red Hat Subscription Manager, but it was tested only on Enterprise Linux.
+endif::[]
+* The host is registered to Katello.
+The utility will exit if the host is registered to Red Hat Subscription Manager directly.
```

Comment:
```suggestion
The utility will not work if the host is registered to Red Hat Subscription Manager directly.
```

I assume "exit" means abort/stop?

Reply:
This sentence is supposed to clarify why we mention Katello instead of {Project}. Does it help?

Reply:
OK, then please reword this. I might create a separate PR to rename Katello in the future. But this is definitely good for now. 👍